src/main/java/ProjectOcean/Controller/SearchBrowseController.java

```diff
@@ -92,7 +94,7 @@ private void filterAndDisplayCourses() {
         filterBasedOnStudyPoints(filteredSearchResult);
         //Displays filtered result
         for(ICourse course : filteredSearchResult) {
-            CourseListIconController courseListIcon = new CourseListIconController(course, model, showDetailedInformationWindow, addIconToScreen);
+            CourseController courseListIcon = new CourseController(course, model, visualFeedback, showDetailedInformationWindow, addIconToScreen);
```

Comment:
Variable name could be mirrored with the class name change.

Reply:
Very good point, fixed.